scipy/signal/tests/test_filter_design.py

```diff
@@ -1643,6 +1645,9 @@ def test_basic(self, xp):
         xp_assert_close_nulp(b_z, xp.asarray(b_zref))
         xp_assert_close_nulp(a_z, xp.asarray(a_zref))
 
+        xp_assert_close_nulp(b_z, xp.asarray(b_zref))
+        xp_assert_close_nulp(a_z, xp.asarray(a_zref))
+
```

Comment:
Oops. This looks like something that went wrong when I was resolving merge conflicts.